diff --git a/webpack/rules/fonts.js b/webpack/rules/fonts.js
index 69e52a0..45df707 100755
--- a/webpack/rules/fonts.js
+++ b/webpack/rules/fonts.js
@@ -1,5 +1,9 @@
 module.exports = {
   test: /\.(eot|svg|ttf|woff|woff2)$/,
   exclude : /(node_modules)/,
-  use     : ['file-loader'],
+  loader  : 'file-loader',
+  options: {
+    outputPath: 'fonts/',
+    publicPath: 'assets/adminator/'
+  },
 };
